tests/unit_tests/audit_log/test_audit_table.py

```diff
@@ -195,5 +194,7 @@ def insert_query(query_string: str, audit: bool):
     expected_matches = []
     for s in samples:
         expected_matches += s[1:]
-    for actual, expected in zip(statements, expected_matches):
-        assert re.match(expected, actual, re.DOTALL)
+    for i, (actual, expected) in enumerate(zip(statements, expected_matches)):
+        matches = re.match(expected, actual, re.DOTALL)
+        LOG.debug(f"{i+1}. {matches and True}")
```

Comment:
is this intendd?

Reply:
It's not essential to the test, but it helped me debugging and fixing.
And I found myself removing and adding this repeatedly. :slightly_smiling_face: